crates/solidity/inputs/language/src/definition.rs

```diff
@@ -6627,37 +6627,366 @@ codegen_language_macros::compile!(Language(
         ),
         BuiltInFunction(name = "assert", parameters = ["bool condition"]),
         BuiltInFunction(
-            name = "require",
-            parameters = ["bool condition"],
-            enabled = From("0.5.0")
+            name = "blockhash",
+            parameters = ["uint blockNumber"],
+            return_type = "bytes32",
+            enabled = From("0.4.22")
+        ),
+        BuiltInFunction(
+            name = "blobhash",
```

Comment:
I see `blobhash` and also `YulBlobHashKeyword` defined here. Should the keyword be removed?

I wonder if we have this bug in other places. e.g. `YulBlockHashKeyword`.

Reply:
Yeah, from what I could gather, most (all?) Yul built-ins are encoded as keywords. As to remove the keywords, I don't know what's the best strategy moving forward. I guess it depends on how we will implement those built-ins in the compilation.

Reply:
Copying context from another thread:

> That makes me wonder more about `YulBuiltInFunction`, and if it should stay in the grammar (useful to users), or just remove it and the keywords there, treating the functions as just regular built-ins?
>
> This is a remaining artifact of the original grammar that we started with, and removing it would clean up/simplify our grammar greatly, as no other language would specify library functions to their grammar/handle it at the lexical stage. Maybe we should just remove them, and treat them like any other built-in function? since they are not even truly reserved, and the list is not comprehensive (for example [`jumpdest`](https://www.evm.codes/#5B) has no corresponding Yul function).
>
> I wonder if we are able to treat it just as any other built-in function. An identifier that gets bound to a function name, and resolved to the specific built-in in binding. This is what we do for Solidity’s built-in functions; we don’t have `SolidityBuiltInFunction` enum in the grammar with variants like `RequireKeyword` and `AssertKeyword`. This simplifies the grammar, removes unnecessary work the parser has to do, and provides the same end result. The versioning info will still be used in binding.
>
> Additionally, relying on specifically the `built_ins` entry in the language definition let's us to also set a `lexical_context` field for specific groups of functions/variables, so that they are bound only from Yul or Solidity.